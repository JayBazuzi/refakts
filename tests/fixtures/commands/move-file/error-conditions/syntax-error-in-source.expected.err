Syntax errors detected in src/broken.ts