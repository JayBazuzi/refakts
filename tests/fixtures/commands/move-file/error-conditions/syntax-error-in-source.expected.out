Warning: Syntax errors detected in src/broken.ts
File moved: src/broken.ts â†’ src/fixed/broken.ts
No import references found to update